lib/stdlib/doc/src/lists.xml

```diff
@@ -365,7 +365,9 @@ flatmap(Fun, List1) ->
       <desc>
         <p>Returns a list containing the sorted elements of the list
           <c><anno>TupleList1</anno></c>. Sorting is performed on the <c><anno>N</anno></c>th
-          element of the tuples. The sort is stable.</p>
+          element of the tuples. The sort is stable and in ascending order.
+          Strings are sorted on the value of underlying codepoints; for Unicode,
+          this may not be alphabetical order.</p>
```

Comment:
I think you should use past tense here. "might not" instead of "may not". "may not" implies permission.